app/src/main/java/com/duckduckgo/app/browser/BrowserWebViewClient.kt

```diff
@@ -213,7 +215,7 @@ class BrowserWebViewClient @Inject constructor(
                 }
             }
         } catch (e: Throwable) {
-            Timber.e(e.localizedMessage)
+            crashLogger.logCrash(CrashLogger.Crash(shortName = "m_webview_should_override", t = e))
```

Comment:
Do we want a different shortName? Is there any value? The stacktrace should point to here anyways right?

Reply:
Probably no big difference at the moment, but I think putting a different `shortName` for captured crashes may have some benefits.

Pixel name will still be `APPLICATION_CRASH_GLOBAL` since we are using `logCrash`.

For the `shortName`, we could also use `APPLICATION_CRASH_GLOBAL`, but I don't think 100% correct for something we've captured. Having different SN allows you to filter or group (again, we are not doing it now, but it could be a usecase in the future). If at some point we decide to remove that pixel from the global crash, and fire it as a pixel, that will be the pixel name, so we can follow the trend.